README.md

```diff
@@ -842,7 +847,10 @@ let try_mkdir path =
 The checks also apply to following symlinks:
 
 ```ocaml
-# Unix.symlink "dir1" "link-to-dir1"; Unix.symlink "/tmp" "link-to-tmp";;
+# Unix.symlink "dir1" "link-to-dir1";
+  Unix.symlink
+    (if Sys.win32 then Sys.getenv "TEMP" else "/tmp")
```

Comment:
Welcome to Windows!

Reply:
We could perhaps use `Filename.get_temp_dir_name ()` here. We don't really care where it points.